yrs/src/doc.rs

```diff
@@ -41,7 +42,7 @@ use std::cell::RefCell;
 pub struct Doc {
     /// A unique client identifier, that's also a unique identifier of current document replica.
     pub client_id: u64,
-    store: RefCell<Store>,
+    store: Rc<UnsafeCell<Store>>,
```

Comment:
For sake of being able to pass transaction to ywasm and yffi I had to remove lifetime parameter from `Transaction<'txn>` (now it's just `Transaction`). To do so, I decided to pass store not by `RefMut` but rather by `Rc`. This allows us to have multiple concurrent transactions open at the same time. In order to be able to modify store we also needed interior mutability - I used `UnsafeCell` rather than `RefCell` as store is accessed very often and `UnsafeCell` overhead is basically 0.